CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+* [BUGFIX] Scrape: Report error if the series contains invalid metric names or labels. #11074
```

Comment:
Please remote the changelog entry ; we generate changelog when releasing

Reply:
removed